Source/App.js

```diff
@@ -1 +1,417 @@
+            });
+        }
+        return property;
+    }
+
+    // Load drone 3D model
+    // Set a starting position in an interesting environment.
+    var position = Cesium.Cartesian3.fromDegrees(-74.01881302800247, 40.72694833660694, 700);
+    var heading = Cesium.Math.toRadians(135);
+    var pitch = 0;
+    var roll = 0;
+    var hpr = new Cesium.HeadingPitchRoll(heading, pitch, roll);
+    var orientation = Cesium.Transforms.headingPitchRollQuaternion(position, hpr);
+
+    position = computeCircularFlight(-73.95881302800247, 40.78694833660694, 0.0005)
+    var drone = viewer.entities.add({
+        //Set the entity availability to the same interval as the simulation time.
+        availability : new Cesium.TimeIntervalCollection([new Cesium.TimeInterval({
+            start : viewer.clock.startTime,
+            stop : viewer.clock.stopTime
+        })]),
+        name : 'drone',
+        position : position,
+        orientation : new Cesium.VelocityOrientationProperty(position),
+        model : {
+            uri : './Source/SampleData/Models/CesiumDrone.gltf',
+            minimumPixelSize : 128,
+            maximumScale : 2000
+        },
+        //Show the path as a line sampled in 100 second increments.
+        path : {
+            resolution : 100,
+            material : new Cesium.PolylineGlowMaterialProperty({
+                glowPower : 0.1,
+                color : Cesium.Color.YELLOW
+            }),
+            width : 10
+        }
+    });
+
+    // Interpolate smoothly between position sample points.
+    drone.position.setInterpolationOptions({
+        interpolationDegree : 2,
+        interpolationAlgorithm : Cesium.HermitePolynomialApproximation
+    });
+
+    //////////////////////////////////////////////////////////////////////////
+    // Custom mouse interaction for highlighting and selecting
+    //////////////////////////////////////////////////////////////////////////
+
+    // If the mouse is over a point of interest, change the entity billboard scale and color
+    var previousPickedEntity = undefined;
+    var handler = new Cesium.ScreenSpaceEventHandler(viewer.scene.canvas);
+    handler.setInputAction(function(movement) {
+        var pickedPrimitive = viewer.scene.pick(movement.endPosition);
+        var pickedEntity = (Cesium.defined(pickedPrimitive)) ? pickedPrimitive.id : undefined;
+        // Unhighlight the previously picked entity
+        if (Cesium.defined(previousPickedEntity)) {
+            previousPickedEntity.billboard.scale = 1.0;
+            previousPickedEntity.billboard.color = Cesium.Color.WHITE;
+        }
+        // Highlight the currently picked entity
+        if (Cesium.defined(pickedEntity) && Cesium.defined(pickedEntity.billboard)) {
+            pickedEntity.billboard.scale = 2.0;
+            pickedEntity.billboard.color = Cesium.Color.ORANGERED;
+            previousPickedEntity = pickedEntity;
+        }
+    }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
```

Comment:
I think it's better to do `LEFT_CLICK` than `MOUSE_MOVE`.  I think doing things on `MOUSE_MOVE` can get annoying